datasets/caner/caner.py

```diff
@@ -90,7 +91,7 @@ def _info(self):
             homepage=_HOMEPAGE,
             license=_LICENSE,
             citation=_CITATION,
-        )
+        task_templates=[TextClassification(text_column="token", label_column="ner_tag")])
```

Comment:
Hmm, this dataset is described as a NER dataset, but the features suggest otherwise (i would expect a `Sequence` feature for `ner_tag` ...)

Reply:
Will fix this separately